crates/mitex/src/lib.rs

```diff
@@ -949,29 +951,35 @@ mod tests {
     fn test_convert_lr() {
         assert_debug_snapshot!(convert_math(r#"$\left.\right.$"#), @r###"
         Ok(
-            "lr()",
+            "lr(  )",
         )
         "###);
         assert_debug_snapshot!(convert_math(r#"$\left.a\right.$"#), @r###"
         Ok(
-            "lr(a )",
+            "lr( a  )",
         )
         "###);
         assert_debug_snapshot!(convert_math(r#"$\alpha\left.\right.$"#), @r###"
         Ok(
-            "alpha lr()",
+            "alpha lr(  )",
```

Comment:
what does this looks like?

Reply:
It would be better if we have snapshot test in svg/png so we can be more confident about convert result 🤔

Reply:
https://github.com/Enter-tainer/typstyle/blob/6b287661f63a6ec8d8b147e3d99682949634a5bb/tests/assets.rs#L162

Reply:
I test it locally and they are the same. We may convert this into a issue